oc-chef-pedant/oc-chef-pedant.gemspec

```diff
@@ -14,6 +14,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency('rspec', '~> 3.2')
   s.add_dependency('activesupport', '>= 4.2.7.1', '< 8.0') # For active_support/concern
+  s.add_dependency('chef-utils', '>= 16.0')
```

Comment:
Technically you depend on 16.17.51 or later right?

Reply:
Correct, would you like me to fix that?